spec/migrations/update_enterprise_instagram_links.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+require Rails.root.join('db/migrate/20221101201147_update_entreprise_instagram_links.rb')
+
+describe UpdateEntrepriseInstagramLinks do
```

Comment:
this file should contains `_spec` at the end of its name (like it's done in the folder `spec/migrations`)